test/index.ts

```diff
@@ -0,0 +1,27 @@
+import 'mocha';
+import { startServer } from '../server';
+import axios from 'axios';
+import { expect } from 'chai';
+
+describe('Array', function () {
+  before('starting server', function (done) {
+    startServer().then(done);
+  });
+  it('should get a hello back from the server', async function () {
+    const endpoint = 'http://localhost:3000/';
```

Comment:
dado que o endpoint vai ser o mesmo para todas as chamadas, o mesmo poderia ser declarado fora do `it`, não?